frontend/public/components/factory/ListPage/ListPageCreate.tsx

```diff
@@ -97,7 +97,9 @@ const ListPageCreate: React.FC<ListPageCreateProps> = ({
   groupVersionKind,
   children,
 }) => {
-  const [k8sModel] = useK8sModel(groupVersionKind);
+  const kind = typeof groupVersionKind !== 'string' ? groupVersionKind.kind : groupVersionKind;
+
+  const [k8sModel] = useK8sModel(kind);
```

Comment:
This is incorrect, we need to be passing the API group and version as well to `useK8sModel`. We need to disambiguate between the same `kind` in different groups as well as different API versions.

Reply:
It looks like `useK8sModel` already takes a `groupVersionKind`, so maybe we don't need to change the code here.

https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/hooks/useK8sModel.ts#L25

If so, it can still be `useK8sModel(groupVersionKind)` as before